examples/plot_theta_square.py

```diff
@@ -74,42 +76,39 @@
         # Calulate hillas parameters
         # It fails for empty pixels   
         try:
-            hillas_params[tel_id] = hillas_parameters(camgeom, cleaned_image)
+           hillas_params[tel_id] = hillas_parameters(camgeom, cleaned_image)
         except:
-            pass
+           pass
 
     if len(hillas_params) < 2: 
-        continue
+           continue
 
-    reco.get_great_circles(hillas_params, event.inst.subarray,
-                           point_azimuth, point_altitude)  
+    reco.get_great_circles(hillas_params,event.inst.subarray,point_azimuth,point_altitude)  
 
     # fit the gamma's direction of origin
     # return reconstructed direction (3 components) with errors on the values
     reco_direction, reco_dir_err = reco.fit_origin_crosses()
-
+    
     # In case fit fails to get any real value
-    if np.isnan(reco_direction).any():
-        continue
+    if np.isnan(reco_direction).any(): continue
 
-    # get angular offset between reconstructed shower direction and MC
-    # generated shower direction
+    # get angular offset between reconstructed shower direction and MC generated shower direction
     off_angle = linalg.angle(reco_direction, shower_direction)
 
     # Appending all estimated off angles
     off_angles.append(off_angle.to(u.deg).value)
 
 # calculate theta square
-thetasq = []
+thetasq=[]
```

Comment:
spaces around assignment operator unless in parameter lists:
https://www.python.org/dev/peps/pep-0008/#other-recommendations